content/en/observability_pipelines/setup_opw.md

```diff
@@ -62,10 +62,22 @@ To set bootstrap options, do one of the following:
 : env var: `DD_OP_THREADS`
 : The number of threads to use for processing (optional, default: the number of available cores).
 
+`proxy`
```

Comment:
Let's make a note that these will work properly on `OPW` versions >= 2.1

Reply:
Added in this [commit](https://github.com/DataDog/documentation/pull/23902/commits/9ee5cfa303027aeb03072376af68c88de4128557).